bigquery/snippets/snippet.go

```diff
@@ -297,6 +404,56 @@ func createTableExplicitSchema(client *bigquery.Client, datasetID, tableID strin
 	return nil
 }
 
+func createTablePartitioned(client *bigquery.Client, datasetID, tableID string) error {
+	ctx := context.Background()
+	// [START bigquery_create_table_partitioned]
```

Comment:
This is the norm for the other snippets, but any particular reason to leave the `ctx` line out of the snippet?

Reply:
The decision predates me, but context construction feels very similar to the client creation. I'll add a comment to the shared include block to demonstrate background context creation as well as client creation.